src/data/news/software_releases.yml

```diff
@@ -1,4 +1,16 @@
-- 
+-
+  software: decred
+  version: 1.8.1
+  Permalink: "https://github.com/decred/decred-binaries/releases/tag/v1.8.1"
+  Params:
+    sortDate: "2023-10-09 16:00:22"
+-
+  software: bison
+  version: 0.1.9
+  Permalink: "https://github.com/companyzero/bisonrelay/releases/tag/v0.1.9"
+  Params:
+    sortDate: "2023-09-28 20:26:50"
```

Comment:
The UTC timestamp I see on GitHub is `2023-09-28 13:00:43`

Reply:
Comment below